scripts/apps/workspace/content/services/ContentService.ts

```diff
@@ -324,7 +324,7 @@ export function ContentService(api, superdesk, templates, desks, packages, archi
             self._fieldsPromise = api.getAll('vocabularies', {
                 where: {
                     $or: [
-                        {field_type: {$in: ['text', 'date', 'media', 'embed', 'urls']}},
+                        {field_type: {$in: ['text', 'date', 'media', 'embed', 'urls', 'custom']}},
```

Comment:
I feel like this array should be inside `../constants`, it's sometimes hard to find the different values a field can have, and `constants` seems like a centralized solution we already have